code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -70,7 +70,6 @@
 	if(SSair.currentpart == SSAIR_ATMOSMACHINERY)
 		SSair.currentrun -= src
 
-	dropContents()
```

Comment:
if a mob's inside a pipe that's destroyed, do they still get kicked out now? that was handled by dropContents()

Reply:
Yes. `/obj/machinery/Destroy()` calls `drop_contents()` (which drops all contents, the snake_case renamed `dropContents()`), making the additional call in the child Destroy proc redundant. Nothing in the child proc appeared to be reliant on drop_contents being called before the parent's Destroy() proc.